### PR TITLE
Missing variables of a Pumped-Storage Hydro unit

### PR DESCRIPTION
**Pull request by erikfilias**
This PR adds the following missing variables:

`Initial Storage Level|Electricity|Hydro|Pumped Storage`
`Maximum Charge|Electricity|Hydro|Pumped Storage`
`Maximum Discharge|Electricity|Hydro|Pumped Storage`

And modifies the description of existent pumped-storage hydro's variables.

### CODE REVIEW

File: nomenclature/definitions/variable/technology/power-plant.yaml [hunk 1]
```diff
@@ -137,20 +137,32 @@ Inflows|Electricity|Hydro|Reservoir:
   description: Inflows into a reservoir expressed in energy
   unit: MWh
 
+Initial Storage Level|Electricity|Hydro|Pumped Storage:
+   description: Initial energy stored in the pumped-storage hydro unit at first load level.
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/technology/power-plant.yaml:141]: What is meant with "at first load level"? Also, for clarification: If this variable is used with hourly timesteps, then the value refers to the stored energy at the beginning of the hour - correct?

File: nomenclature/definitions/variable/technology/power-plant.yaml [hunk 2]
```diff
@@ -137,20 +137,32 @@ Inflows|Electricity|Hydro|Reservoir:
...
+   unit: GWh
+
+Maximum Charge|Electricity|Hydro|Pumped Storage:
+   description: Maximum charge when storing energy the pumped-storage hydro unit.
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/technology/power-plant.yaml:145]: Maybe missing "in"?

**Review by danielhuppmann — CHANGES_REQUESTED** [review]
Thanks @erikfilias, see comments and requests for clarification inline.

File: nomenclature/definitions/variable/technology/power-plant.yaml (hunk 1 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/technology/power-plant.yaml:141]: Right, this variable is used at the beginning of the first hour into hourly timesteps.
I'll change it.

File: nomenclature/definitions/variable/technology/power-plant.yaml (hunk 2 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/technology/power-plant.yaml]: I fix it.

**Review by danielhuppmann — APPROVED** [review]
Thanks @erikfilias

**Pull request merged**